src/Raven.Server/Documents/Handlers/Debugging/StorageHandler.cs

```diff
@@ -324,5 +324,32 @@ public async Task ScratchBufferPoolInfoReport()
             using (var processor = new StorageHandlerProcessorForGetScratchBufferReport(this))
                 await processor.ExecuteAsync();
         }
+        
+        [RavenAction("/databases/*/debug/storage/environment/free-space-snapshot", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetFreeSpaceInfo()
```

Comment:
Shouldn't this be an admin endpoint?

Reply:
I didn't see any sensitive information here and I mimicked the other endpoint in this handler.
Why should it be an admin endpoint?

Reply:
Okay, good point then